### DISCUSSION TITLE
USB support worse than ever. kmod loads; device never shows.

### DISCUSSION THREAD

**Discussion by ballo** [utmapp/UTM#6071] [Linux]
**BEFORE SUBMITTING YOUR ISSUE, PLEASE LOOK AT THE PINNED ISSUES AND USE THE SEARCH FUNCTION TO MAKE SURE IT IS NOT ALREADY REPORTED. ALWAYS COMMENT ON AN EXISTING ISSUE INSTEAD OF MAKING A NEW ONE.**

**Describe the issue**  
If I connect a USB device (in this case a Atheros wifi modem), the kmod loads but there is no device. Previously, the device would load but it would be unusable. I can only guess that the data is being corrupted. Mac is reporting high CPU use as well but I'm unsure if that's related.

**Configuration**  
* UTM Version: 4.4.5
* macOS Version: Ventura
* Mac Chip (Intel, M1, ...): M2 Max (previous attempt on 8 core M1)

**Debug log**  
I can't open VM settings while the VM is running. Is this a design flaw? The debug log in the VM bundle just shows a line of upside-down question marks 

Anyway, download any Linux distro, plug in Atheros USB modem, and I'm guessing you'll get the same failure. If I have time tomorrow I might try a realtek modem.

**Comment by ballo** [comment]
Oh great, now it refuses to attach the USB device at all (still shows up in the menu). Doesn't matter which VM. Tried restarting UTM and it crashed. I'll try to embed the crash report. I'll probably have to reboot my Mac or something at this point. 

Does USB work for anyone? I've had nothing but failure of one sort or another; never success. 

[crash.txt](https://github.com/utmapp/UTM/files/14100396/crash.txt)

**Reply by bewl87** [reply]
i'm having usb issues too, shows in task bar but doesn't show up in explorer

**Comment by bluesky4eagles** [comment]
hi. I gome same issue with usb attachments , its showing the divice connected to machine UTM - unfortunately the mashine dont give the permission to open it ! Running’s on mac
